docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/05 - Visits.md

```diff
@@ -0,0 +1,44 @@
+# Visits
+
+## Overview
+The **Visit** designates the instance in time where a candidate has visited or contacted a site and underwent any sort of data collection.
+
+**Visit** and **Timepoint** are terms used interchangeably in LORIS. **Visits** are defined in the `Visit_Windows` table of the database.
+
+## Adding Options
+
+### Front End
+ _not yet available_
+
+### SQL (Recommended)
+
+This table must be populated with visit labels - the Imaging Pipeline critically depends on this.  To populate with visit labels, run tools/populate_visit_windows.php - or manually insert study-specific information:
+
+```sql
+INSERT INTO Visit_Windows (Visit_label,  WindowMinDays, WindowMaxDays, OptimumMinDays, OptimumMaxDays, WindowMidpointDays) VALUES ('V1', '0', '100', '40', '60', '50');
+```
+
+If age is a not critical factor in study visit scheduling, define Min value as 0, and Max value as 2147483647 (this is the maximum integer size).
```

Comment:
which `Min` value(s)?

Reply:
also might be best to just say "maximum integer size" rather than provide a specific value. column type could change or size could be variable across different SQLs

Reply:
right on all counts

Reply:
couldn't those field be simply set to `NULL`? Then it is clear that there is no restriction on the age?

Also, it might be good to explain what each of those fields stand for or are supposed to represent as I am not sure what is the difference between  `WindowMinDays ` and `OptimumMinDays ` for example. I would expect that someone new to LORIS would not know either.

Reply:
will set them all to null here 

As for the specific explanation of the fields, same as for other PRs, I'm not comfortable adding them in here as they will most probably not be maintained. either a separate document or a front end would have that